Makefile

```diff
@@ -293,10 +295,186 @@ docker-push-magpie: docker-build-magpie		## push only built docker image for Mag
 .PHONY: docker-push
 docker-push: docker-push-magpie docker-push-adapter	 ## push built docker images for Magpie application and MagpieAdapter for Twitcher
 
-## Conda targets
+DOCKER_TEST_COMPOSES := -f "$(APP_ROOT)/ci/docker-compose.smoke-test.yml"
+.PHONY: docker-test
+docker-test: docker-build-magpie	## execute a smoke test of the built image for Magpie application (validate that it boots)
+	@echo "Smoke test of built application docker image"
+	docker-compose $(DOCKER_TEST_COMPOSES) up -d
+	sleep 5
+	curl localhost:2001 | grep "Magpie Administration"
+	docker-compose $(DOCKER_TEST_COMPOSES) stop
```

Comment:
docker-compose down -v instead of stop?  So it doesn't leave not running containers behind.

Reply:
Ah I see there's the `docker-clean` target that will do a `docker-compose down`.  Why not just down right in here so the next invocation of target `docker-test` starts fresh?

Reply:
I don't need to re-pull all the other things over again. 
Magpie is rebuilt so the tag will invalidate the old and use the fresh one.
The targets combine as few operations as possible to allow using them not only in travis-ci.

Reply:
Not sure what you mean by "re-pull all the other thing".  All parents layers are cached on first run, on subsequent runs, nothing new should be pulling.

But I am fine you keep this as is.  Hopefully the user remember to call `make docker-clean`.